doc/classes/Dictionary.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Dictionary type. Associative container which contains values referenced by unique keys. Dictionaries are always passed by reference.
+		Erasing elements while iterating over them [b]is not supported[/b]. Godot does not print any warning but stops iterating silently, which is not supposed logic anyway.
```

Comment:
What do you mean by "which is not supposed logic anyway"?

Reply:
It means that one actually can erase elements during iteration over dictionary, and when erase the first element, the loop is interrupted immediately. This logic works this way now, but is not supposed for using in project code, however.

Reply:
I think it's better to remove the last sentence then, and stop at "not supported". No need to document *how* it behaves in the unsupported case, as it might indeed change in the future.

Reply:
Ok, changed.